lang/scala/scala.talon

```diff
@@ -24,15 +24,19 @@ settings():
     user.code_protected_variable_formatter = "PRIVATE_CAMEL_CASE"
     user.code_public_variable_formatter = "PRIVATE_CAMEL_CASE"
 
-state package: "package "
+[state] {user.scala_modifier}:
```

Comment:
I think I'd keep the state prefix required for these ones, we tend to stay away from single word commands in knauj public to reduce the chance of collisions with future commands and also because they tend to recognise less well.

Reply:
Good point, thank you! I amended this in the commit.